support-frontend/assets/helpers/redux/user/thunks.ts

```diff
@@ -14,11 +16,24 @@ export const getRecurringContributorStatus = createAsyncThunk<
 >(
 	'user/getRecurringContributorStatus',
 	async function getIsRecurringContributor() {
+		const authWithOkta = isSwitchOn('featureSwitches.authenticateWithOkta');
+		const accessToken = cookie.get('GU_ACCESS_TOKEN');
+
+		// No point in making the call if we don't have an access token as we know it's going to fail
+		if (authWithOkta && !accessToken) return {};
+
+		// Okta authorization uses the Authorization header rather than a cookie
+		const authHeader =
+			authWithOkta && accessToken
+				? { Authorization: `Bearer ${accessToken}` }
+				: undefined;
```

Comment:
Could maybe extract this into a function and use it here and above?

Reply:
Yes I'll do that.

Reply:
Extracted function in a2e2445e95effdeb9f8a54fc90187148c3bd312d